### PR TITLE
Add a container model object

### PR DESCRIPTION
**Pull request by Anthchirp**
Bit of an odd one as it currently only works when populated via a `DataCollectionGroup`.
Only exposes three properties at this time.

Updated history file for 5.6.1 release later today.

### CODE REVIEW

File: ispyb/model/container.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        """Load/update information from the database."""
+        raise NotImplementedError("Loading containers is not currently supported")
+
+    @property
+    def containerid(self):
+        """Returns the ID associated with this container information."""
+        return self._containerid
+
+    def __bool__(self):
+        """Container object evaluates to True in a boolean context if grid
```
Comment (KarlLevik) [inline ispyb/model/container.py:37]: Grid information? That sounds a bit strange for containers?

**Review by KarlLevik — APPROVED** [review]
That looks fine to me, though I'm not sure about the comment re: grid info. Only exposing the relevant container properties is def. a good idea. Do you need the email address as well, though, in case you need to send an email notification?

**Review by KarlLevik — APPROVED** [review]
:+1:

File: ispyb/model/container.py (hunk 1 above)
Comment (Anthchirp) [reply ispyb/model/container.py:37]: only saw this now. fixed in 4bb6496b534f3de3742a8545196b6eb47a0fddbe

**Pull request merged**